### ISSUE TITLE
Segmentation Model

### ISSUE THREAD

**Issue by chenye2001** [Gait3D/Gait3D-Benchmark#32]
Thank you for releasing the Gait3D dataset, it is an incredibly valuable contribution! I have a few questions about your work and would greatly appreciate your clarification:

1. Could you please specify which version or implementation of the HR-Net segmentation model was used in your work? Was it based on the official HR-Net segmentation repository?

2. Did you fine-tune the segmentation model for your specific use case? If so, could you share some details about the fine-tuning process, such as the dataset and hyperparameters?

Thank you again for your contributions. I look forward to your response!

**Comment by JinkaiZheng** [comment]
Thanks for your question! 

1. We used the version of seg_hrnet_w48 in the official HR-Net segmentation repository.

2. Yes, to ensure the segmentation quality, we have labeled the segmentation labels in the scene and fine-tuned the model. However, the relevant dataset and model cannot be open source for some reason. We appreciate your understanding.

**Comment by chenye2001** [comment]
Thank you for your detailed and quick response!
